choria/connection.go

```diff
@@ -583,15 +586,31 @@ func (conn *Connection) Connect(ctx context.Context) (err error) {
 		}),
 	}
 
-	if !(conn.config.DisableTLS || conn.choria.ShouldUseNGS()) {
+	switch {
+	case conn.config.Choria.ClientAnonTLS && !conn.config.InitiatedByServer:
+		conn.logger.Debug("Setting anonymous TLS for NATS connection")
+
+		cfg := tlssetup.TLSConfig(conn.config)
+		tlsc := &tls.Config{
+			MinVersion:               tls.VersionTLS12,
+			PreferServerCipherSuites: true,
+			CipherSuites:             cfg.CipherSuites,
+			CurvePreferences:         cfg.CurvePreferences,
+			InsecureSkipVerify:       true,
```

Comment:
The intention is to accept any client cert?

Reply:
Yes, this would be a extra choria broker whose purpose is specifically to accept clients using the AAA server - who don’t have certs at all. 

This lets them connect without lowering the security of the existing choria network. 

Combined with the setting to disable servers and limiting clients from known locations this will be a safe way for clients only. I might require that those 2 settings are configured before enabling this mode

More background in slack and the issue.